lunes_cms/cms/admins/user_admin.py

```diff
@@ -0,0 +1,7 @@
+from django.contrib import admin
+
+from ..models import RegisterUserForm
+
+
+class UserAdmin(admin.ModelAdmin):
```

Comment:
New admins also have to be registered in [lunes_cms/cms/admin.py](https://github.com/digitalfabrik/lunes-cms/blob/develop/lunes_cms/cms/admin.py), otherwise they are just a few lines of code that are never executed.